### DISCUSSION TITLE
What is the Latest Version of SQLiteStudio that Can Run on Windows XP SP3?

### DISCUSSION THREAD

**Discussion by spaceman5** [pawelsalawa/sqlitestudio#4304] [Q&A]
Hi

I am using Windows XP SP3, and downloaded sqlitestudio-3.3.3.zip.

It does not run on XP SP3, so I will download an older version..
Can you please tell me what is the latest version that will run on XP SP3?

Thank you very much

**Comment by pawelsalawa** [comment]
Version 3.2.1 (https://github.com/pawelsalawa/sqlitestudio/releases/tag/3.2.1) should work on XP.

**Comment by juanitogan** [comment]
3.2.1 errors with: "The procedural entry point CancelIoEx could not be located in the dynamic link library KERNEL32.dll."

Also, 3.2.1 is built with Qt 5.10, which is does not support XP.

I tested several versions and none of these worked on XP (each with a different error message):
- 3.4.4
- 3.3.3
- 3.2.1
- 3.1.1
- 3.0.7
- 2.99.6

The latest version I could find that works on XP (SP3 32-bit) is:
- 2.1.5

3.1.1 uses Qt 5.7, which appears to unofficially work with XP (depending on features needed), so it _might_ run on XP if it were compiled with a target of XP.  Qt 5.6, however, was the last Qt version that officially supported XP.

3.0.7 uses Qt 5.4.  Thus, in theory, 3.0.7 might be the last version that can be compiled for XP without modification (assuming Qt is the only dependency that is a limiting factor here).
